airflow/example_dags/example_custom_weight.py

```diff
@@ -0,0 +1,65 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Example DAG demonstrating the usage of the CustomWeightRule."""
+
+from __future__ import annotations
+
+import datetime
+
+import pendulum
+# [START example_custom_weight_dag]
+from airflow.models.dag import DAG
+from airflow.providers.standard.operators.bash import BashOperator
+from airflow.providers.standard.operators.empty import EmptyOperator
+from decreasing_priority_weight_strategy import DecreasingPriorityStrategy
```

Comment:
Just checked, the import directly from plugins is not working, in the examples it is working if you make it an absolute path (as load-able via the airflow package)
```suggestion
from airflow.example_dags.plugins.decreasing_priority_weight_strategy import DecreasingPriorityStrategy
```